packages/plugin-core/src/_extension.ts

```diff
@@ -848,6 +860,44 @@ export async function showLapsedUserMessage(assetUri: vscode.Uri) {
     });
 }
 
+export function shouldDisplayMissingDefaultConfigMessage(opts: {
```

Comment:
Done for the missing default prompt logics.

Created a task [[Move Activation Prompt Logic to Separate Utility Class|dendron://private/task.2022.03.28.move-activation-prompt-logic-to-separate-utility-class]]

to follow up with the rest of the prompt logic since this involves fixing the relevant tests which will be slightly out of scope for this PR